Tools/scripts/smelly.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# Script checking if all symbols exported by libpython start with "Py" or "_Py"
+
+import subprocess
+import sys
+import sysconfig
+
+
+def get_exported_symbols():
+    LIBRARY = sysconfig.get_config_var('LIBRARY')
+    if not LIBRARY:
+        raise Exception("failed to get LIBRARY")
+
+    args = ('nm', '-p', LIBRARY)
+    print("+ %s" % ' '.join(args))
+    proc = subprocess.run(args, stdout=subprocess.PIPE, universal_newlines=True)
+    if proc.returncode:
+        sys.stdout.write(proc.stdout)
+        sys.exit(proc.returncode)
+
+    stdout = proc.stdout.rstrip()
+    if not stdout:
+        raise Exception("command output is empty")
+    return stdout
+
+
+def get_smelly_symbols(stdout):
```

Comment:
`stdout` is not a very informative name here, and somewhat confusing in the body of the function.

Reply:
Done, renamed to nm_output.